student.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'person'
+class Student < Person
+  def initialize(classroom)
+    super(age, name, parent_permission)
```

Comment:
- [ ] Great job working on the classes so far but I think since you used `age, name, parent_permission` you should pass them in th constructor as parameters as well with their initial values as well so it would be somthing like this  ⬇💯


>  def initialize(age, classroom, name = "Unknown", parent_permission = true)